index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=pokemon-cards', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('SELECT * FROM `pokemon-card`;');
+
+$query->execute();
+
+$allPokemon = $query->fetchAll();
+
+require_once 'functions.php';
```

Comment:
put this at the top before PDO stuff (this is just convention)

Reply:
Done